pkg/pillar/cmd/volumemgr/handlevolume.go

```diff
@@ -16,7 +19,73 @@ func handleVolumeCreate(ctxArg interface{}, key string,
 	log.Infof("handleVolumeCreate(%s)", key)
 	config := configArg.(types.VolumeConfig)
 	ctx := ctxArg.(*volumemgrContext)
-	updateVolume(ctx, config)
+	status := lookupVolumeStatus(ctx, config.Key())
+	if status != nil {
+		log.Fatalf("status exists at handleVolumeCreate for %s", config.Key())
+	}
+	status = &types.VolumeStatus{
+		VolumeID:                config.VolumeID,
+		ContentID:               config.ContentID,
+		VolumeContentOriginType: config.VolumeContentOriginType,
+		MaxVolSize:              config.MaxVolSize,
+		ReadOnly:                config.ReadOnly,
+		GenerationCounter:       config.GenerationCounter,
+		VolumeDir:               config.VolumeDir,
+		DisplayName:             config.DisplayName,
+		RefCount:                config.RefCount,
+		LastUse:                 time.Now(),
+	}
+	publishVolumeStatus(ctx, status)
+	if info, err := os.Stat(status.PathName()); err == nil {
+		ctConfig := lookupContentTreeConfig(ctx, status.ContentID.String())
+		if ctConfig == nil {
+			log.Fatalf("Attached content tree config %s doesn't exist at handleVolumeCreate for %s",
+				config.ContentID.String(), config.Key())
+		}
```

Comment:
Why is this a fatal? A device which gets upgraded from a prior EVE version then the upgradeconverter in my PR will make the volume appear at the pathname, yet these is no content tree.

When there is no CR we don't know the format (unless we want to guess that is it a CONTAINER if PathName is a directory,